src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -32,7 +32,8 @@ import { CommandProcessExecutionFactory } from "../commandProcessExecution";
 import { PythonBridge, PythonException } from "python-bridge";
 import * as path from "path";
 import { DBTProject } from "../manifest/dbtProject";
-import { existsSync } from "fs";
+import { existsSync, readFileSync } from "fs";
+import * as yaml from "js-yaml";
```

Comment:
We are already using yaml library to read yaml in other places. let's not introduce another library for the sake of it.